### PR TITLE
PR-MSH-USERSTATUS

### LINKED ISSUE #30: UserStatus
Implement core CURD functionality 
- APIs
- Services
- DBRepository
- Store Procedures
- Unit tests

### PR DESCRIPTION
**Pull request by MariamShaikh**
#30 ready to go!

### CODE REVIEW

File: src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs [hunk 1]
```diff
@@ -65,5 +95,140 @@ public void UserStatusCreate_InValid()
             Assert.IsTrue(condition: ErrorCode.ADDERROR == response.Value.ToString());
```
Comment (maazk9119) [inline src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs]: @MariamShaikh When you comparing values must have to use AreEqual. IsTrue only use for asserting Boolean values.

File: src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs [hunk 2]
```diff
@@ -65,5 +95,140 @@ public void UserStatusCreate_InValid()
...
+            var updatedUserStatus = new UserStatus()
+            {
+                RecId = id,
+                Status = "Not Approved"
+            };
+
+            var response = _controller.Update(updatedUserStatus) as NoContentResult;
+
+            //Assert
+            Assert.IsTrue(condition: HttpStatusCode.NoContent == (HttpStatusCode)response.StatusCode);
```
Comment (maazk9119) [inline src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs]: Same - fix it as well.

File: src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs (hunk 1 above)
Comment (MariamShaikh) [reply src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs]: fixed

File: src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs (hunk 2 above)
Comment (MariamShaikh) [reply src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs]: @maazk9119 fixed.

**Review by maazk9119 — APPROVED** [review]
Thanks for resolving the comments - Ready to go!

**Pull request merged**